liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -61,6 +62,7 @@
     private File binaryScanner;
 
     protected static final String PLUGIN_ADDED_FEATURES_FILE = "configDropins/overrides/liberty-plugin-added-features.xml";
+    protected static final String FEATURES_FILE_MESSAGE = "Features have been generated in " + PLUGIN_ADDED_FEATURES_FILE;
```

Comment:
Another note: the message that we add here should be something that we don't change much (if at all) between releases.  Otherwise `removeGenerationCommentFromConfig` would not work on a previously written version of the message.

Reply:
It is kind of a placeholder message but I will improve it. I hope we can get all of our messages in one file so it could one day be translated.